### ISSUE TITLE
Not seeing the search bar

### ISSUE THREAD

**Issue by robmarkcole** [microsoft/satellite-imagery-labeling-tool#30]
I've tried Chrome, Safari and Edge browser, using a python server to access index.html via `python3 -m http.server 8080` - however on none of these do I see the search bar. Perhaps it requires additional config?

**Comment by robmarkcole** [comment]
I guess this requires the Azure Maps functionality enabled

**Comment by rbrundritt** [comment]
Correct. This feature only appears when Azure Maps is enabled (an Azure Maps key must be specified in the map config). We disabled the Azure Maps capabilities in the free public hosted version that is in this github project by default due to abuse.
